src/org/javarosa/core/model/DataType.java

```diff
@@ -38,23 +38,23 @@
  * The model data types.
  */
 public enum DataType {
-    UNSUPPORTED (DATATYPE_UNSUPPORTED),
-    NULL        (DATATYPE_NULL),
-    TEXT        (DATATYPE_TEXT),
-    INTEGER     (DATATYPE_INTEGER),
-    DECIMAL     (DATATYPE_DECIMAL),
-    DATE        (DATATYPE_DATE),
-    TIME        (DATATYPE_TIME),
-    DATE_TIME   (DATATYPE_DATE_TIME),
-    CHOICE      (DATATYPE_CHOICE),
-    CHOICE_LIST (DATATYPE_CHOICE_LIST),
-    BOOLEAN     (DATATYPE_BOOLEAN),
-    GEOPOINT    (DATATYPE_GEOPOINT),
-    BARCODE     (DATATYPE_BARCODE),
-    BINARY      (DATATYPE_BINARY),
-    LONG        (DATATYPE_LONG),
-    GEOSHAPE    (DATATYPE_GEOSHAPE),
-    GEOTRACE    (DATATYPE_GEOTRACE);
+    UNSUPPORTED     (DATATYPE_UNSUPPORTED),
+    NULL            (DATATYPE_NULL),
+    TEXT            (DATATYPE_TEXT),
+    INTEGER         (DATATYPE_INTEGER),
+    DECIMAL         (DATATYPE_DECIMAL),
+    DATE            (DATATYPE_DATE),
+    TIME            (DATATYPE_TIME),
+    DATE_TIME       (DATATYPE_DATE_TIME),
+    CHOICE          (DATATYPE_CHOICE),
+    MULTIPLE_ITEMS  (DATATYPE_MULTIPLE_ITEMS),
```

Comment:
Good. Thanks for knowing that you needed to keep this enum in sync with Constants. And extra special thanks for preserving the beautifully aligned columns that I so lovingly created.

Reply:
😄